src/utils.js

```diff
@@ -3,6 +3,7 @@ import { default as config } from './config.js';
 import axios from 'axios';
 
 const moment = require('moment');
+const variableEnvironments = require('./env.json');
```

Comment:
is there a mechanism in javascript that when you require `config.js` we merge values from env.json with config value? so env.json is hidden from perspective of other components